### PR TITLE
Issue 34

### PR DESCRIPTION
**Pull request by msuhovec**
#34

### CODE REVIEW

File: tasks/fedora-to-rof [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+pid_file="${JOBPATH}/fedora-pids"
+
+#if no pids provided, error and exit
+if [ ! -e "$pid_file" ]; then
+    echo "No fedora-pids file in job directory"
+    exit 1
+fi
+
+# Iterate through pids- use rof fedor_to_rof to retrieve from fedora and generate rof
+for pid  in `jq '.[]' < $pid_file`; do
```
Comment (dbrower) [inline tasks/fedora-to-rof]: Prefer using `$( )` instead of backticks.

File: tasks/fedora-to-rof [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+  this_pid=`echo $pid | sed 's/"//g'`
+  rof_file=`echo ${this_pid} | sed 's/und://'`
```
Comment (dbrower) [inline tasks/fedora-to-rof]: Use `$()` instead of backticks on both of these lines.

File: tasks/fedora-to-rof [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+#if no pids provided, error and exit
+if [ ! -e "$pid_file" ]; then
+    echo "No fedora-pids file in job directory"
+    exit 1
+fi
+
+# Iterate through pids- use rof fedor_to_rof to retrieve from fedora and generate rof
+for pid  in `jq '.[]' < $pid_file`; do
+
+  this_pid=`echo $pid | sed 's/"//g'`
```
Comment (dbrower) [inline tasks/fedora-to-rof]: I think you can get rid of this line if you use the option `--raw-output` with `jq`.

File: tasks/fedora-to-rof (hunk 2 above)
Comment (dbrower) [inline tasks/fedora-to-rof]: If we are using jq version 1.5 in production you can get rid of this line using something like `.[] | sub("und:", "")`

Comment (dbrower) [inline tasks/fedora-to-rof]: nevermind. production is using jq version 1.3.

File: tasks/fedora-to-rof (hunk 1 above)
Comment (msuhovec) [inline tasks/fedora-to-rof]: OK

File: tasks/fedora-to-rof (hunk 2 above)
Comment (msuhovec) [inline tasks/fedora-to-rof]: OK

File: tasks/fedora-to-rof (hunk 3 above)
Comment (msuhovec) [inline tasks/fedora-to-rof]: You know,I've even used that before-- my retention is shot.

File: tasks/fedora-to-rof (hunk 2 above)
Comment (msuhovec) [inline tasks/fedora-to-rof]: Alas, we are not.

**Pull request merged**